code/modules/client/preference_setup/loadout/items/factions.dm

```diff
@@ -141,6 +141,21 @@
 	zavodskoi_headwear["zavodskoi pilotka cap, alt"] = /obj/item/clothing/head/sidecap/zavod/alt
 	gear_tweaks += new /datum/gear_tweak/path(zavodskoi_headwear)
 
+/datum/gear/faction/falerio_headwear
```

Comment:
these should be in the zavod hat selection, same as to how the uniforms are in zavod uniforms selection

Reply:
I think the  reason I didn't was because we don't have a sec-only headgear selection for zavod, but I can double check.

Reply:
As an update, yeah, it doesn't look like we have one for officer only which is why I put it there. Was requested it's open to sec only

Reply:
makes sense in that case